bigquery/tests/unit/test_client.py

```diff
@@ -5161,3 +5161,84 @@ def test__do_multipart_upload_wrong_size(self):
 
         with pytest.raises(ValueError):
             client._do_multipart_upload(file_obj, {}, file_obj_len + 1, None)
+
+    def test__schema_from_json(self):
```

Comment:
Nit: since the function name is `schema_from_json`, the test name should be `test_schema_from_json` (just one underscore between `test` and `schema`.)

Ditto for `schema_to_json` test function name.

Reply:
Updated test names appropriately.